packages/gatsby-transformer-raml/src/utils/resource/examples-to-array.js

```diff
@@ -0,0 +1,11 @@
+function examplesToArray(examples) {
```

Comment:
Are you sure this does not already have an implementation available in the codebase, maybe a more abstract one or in the types code?